.gitignore

```diff
@@ -1,5 +1,8 @@
 # Generated by our build system to indicate the version of the source package
 build/SourceVersion
 
+# Generated by our build system
+doc/sphinx/html/mathjax/
```

Comment:
I can in fact drop this line, because we no longer generate this directory. I may have tested this incorrectly before, but I thought the docs were always build in the source directory. That definitely is the case now (`_build` directory in `doc/sphinx`). What would be the preferred location for outputting the build?

Reply:
Yes. I typically create a clean directory (outside of the source tree), and then run something like
```
cmake ../src
make
```
Up to now, this would not create or change any files in the source directory. Is that also the case after your changes?

Reply:
Sphinx now always builds in the `_build` directory, which is in the sources. Furthermore, one of the custom extensions may add include directives to the tool pages if they are not there yet.

I will look into having Sphinx build in another directory, relative to the root CMake call.

Reply:
You can use ${CMAKE_CURRENT_BINARY_DIR} to get a path to the out-of-source build directory relative to the CMakeLists.txt file (in Thomas' case ../src/doc/). Then the sphinx build directory would be ${CMAKE_CURRENT_BINARY_DIR}/_build.

Reply:
That's exactly what I did now, @mlaveaux. Except I renamed the directory to `sphinx` instead of `_build`, that makes it easier to identify what it is (and CMake already created a `doc` directory). Just have to correct the custom Python code, which still writes to `_build`.

Reply:
Fixed in b03067e778612b01daefc9c944ec7c993a78fb95 and 9ebcdcca3652c1b88707fe38f944a619be81e5e3.